test/unit/overlay.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+var fixtures = require('../fixtures');
+var fs = require('fs');
+var sharp = require('../../index');
+
+sharp.cache(0);
+
+// Main
+describe('Overlays', function() {
+  it('Overlay transparent PNG on solid background', function(done) {
+    sharp(fixtures.inputPngOverlayLayer0)
+      .overlayWith(fixtures.inputPngOverlayLayer1)
+      .toBuffer(function (error, data, info) {
+        if (error) return done(error);
```

Comment:
Thanks for pointing this out, @aseemk. It makes sense in hindsight as the tests previously did work :smile: